server/base/src/main/java/org/apache/accumulo/server/conf/store/impl/ZooPropStore.java

```diff
@@ -438,6 +443,9 @@ private VersionedProperties readPropsFromZk(PropStoreKey<?> propStoreKey)
     try {
       Stat stat = new Stat();
       byte[] bytes = zrw.getData(propStoreKey.getPath(), stat);
+      if (bytes.length == 0) {
```

Comment:
Same comment/question about bytes being null as above.

Reply:
Addressed in ac8d0b8da8 and see previous comment